Consumer/MessageConsumer.php

```diff
@@ -26,14 +29,27 @@ class MessageConsumer implements ConsumerInterface
     /**
      * Message Manager
      *
-     * @var MessageManager
+     * @var MessageDispatcher
      */
-    private $messageManager;
+    private $messageDispatcher;
 
+    /**
+     * Serializer
+     *
+     * @var SerializerInterface
+     */
+    private $serializer;
 
-    public function __construct(MessageManager $messageManager)
+    /**
+     * Constructor
+     *
+     * @param MessageDispatcher   $messageManager
+     * @param SerializerInterface $serializer
+     */
+    public function __construct(MessageDispatcher $messageManager, SerializerInterface $serializer)
```

Comment:
should rename all instances of $messageManager with $messageDispatcher just for naming consistency.